src/wp-includes/class-wp-theme-json.php

```diff
@@ -3236,7 +3262,8 @@ public function merge( $incoming ) {
 		 * some values provide exceptions, namely style values that are
 		 * objects and represent unique definitions for the style.
 		 */
-		$style_nodes = static::get_styles_block_nodes();
+		$style_options = array( 'include_node_paths_only' => true );
+		$style_nodes   = static::get_block_nodes( $this->theme_json, array(), $style_options );
```

Comment:
No need to have a variable for this because it's only used in one place.
```suggestion
		$style_nodes = static::get_block_nodes(
			$this->theme_json,
			array(),
			array( 'include_node_paths_only' => true )
		);
```